src/python/foglamp/translators/omf_translator.py

```diff
@@ -650,54 +621,48 @@ def send_omf_message_to_end_point(message_type, omf_data):
 
     """
 
-    global _log
-
-    try:
-        msg_header = {'producertoken': _producer_token,
-                      'messagetype':   message_type,
-                      'action':        'create',
-                      'messageformat': 'JSON',
-                      'omfversion':    '1.0'}
-
-        response = requests.post(_relay_url, headers=msg_header, data=json.dumps(omf_data), verify=False, timeout=30)
+    max_retry = 5
+    sleep_time = 1
 
-        debug_msg_write("INFO", "Response |{0}| message: |{1}| |{2}| ".format(message_type,
-                                                                              response.status_code,
-                                                                              response.text))
+    status = 0
+    retry = 1
+    loop_continue = True
+    message = ""
 
-    except Exception as e:
-        message = _message_list["e000007"].format(e)
+    while loop_continue:
+        try:
+            status = 0
+            msg_header = {'producertoken': _producer_token,
+                          'messagetype': message_type,
+                          'action': 'create',
+                          'messageformat': 'JSON',
+                          'omfversion': '1.0'}
 
-        _log.error(message)
-        raise Exception(message)
+            response = requests.post(_relay_url, headers=msg_header, data=json.dumps(omf_data), verify=False,
+                                     timeout=30)
 
+            _logger.debug("Response |{0}| message: |{1}| |{2}| ".format(message_type,
+                                                                        response.status_code,
+                                                                        response.text))
 
-def setup_logger():
-    """Configures the log mechanism
+        except Exception as e:
+            message = _message_list["e000007"].format(e)
+            status = 1
 
-    Raises:
-        Exception: Fails to configure the log
+        if status == 0:
+            loop_continue = False
 
-    Todo:
-        # TODO FOGL-203 Configure Python Logging
+        elif retry < max_retry:
 
-    """
-
-    global _log
+            time.sleep(sleep_time)
+            retry += 1
 
-    try:
-        _log = logging.getLogger(_module_name)
-
-        _log.setLevel(logging.DEBUG)
-        handler = logging.handlers.SysLogHandler(address='/dev/log')  # /var/run/syslog
-
-        formatter = logging.Formatter('%(module)s.%(funcName)s: %(message)s')
-        handler.setFormatter(formatter)
-
-        _log.addHandler(handler)
+        else:
+            loop_continue = False
 
-    except Exception as e:
-        raise Exception(_message_list["e000005"].format(e))
+    if status != 0:
+        _logger.warning(message)
+        raise Exception(message)
 
```

Comment:
Code is running but not efficient, We need to improve this piece of code and remove un-necessary code and variables (loop_continue, status, max_retry):
   ```
   num_retry = 1 
   while num_retry < 5:
        try:
            do_something()
            break
        except Exception as e:
            sleep()
            num_retry += 1
```